src/components/Zoo.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { shallowEqual, useSelector } from 'react-redux';
+import Animal from './Animal';
+import Landing from './landing';
+
+const Zoo = () => {
+  const animals = useSelector((state) => state.animals, shallowEqual);
+
+  return (
+    <div className="Zoo">
+      <Landing />
+      <ul className="zooanimals">
+        {animals.map((animal) => (
+          <Animal
+            key={animal.id}
+            animal={animal}
+            name={animal.name}
+            latinName={animal.latin_name}
+            animalType={animal.animal_type}
+            activeTime={animal.active_time}
+            minLength={animal.length_min}
+            maxLength={animal.length_max}
+            minWeight={animal.weight_min}
+            maxWeight={animal.weight_max}
+            lifeSpan={animal.lifespan}
+            habitats={animal.habitat}
+            diet={animal.diet}
+            locattions={animal.geo_range}
+            image={animal.image_link}
+          />
+
+        ))}
```

Comment:
- [ ] According to the design we have alternating lighter and darker colors for the contents on the homepage. Kindly implement that.

  Your Implementation | Given Design
  :---:|:---:
  ![nemwel-alternating](https://user-images.githubusercontent.com/36542986/164277701-7fea9dc1-0edd-4eb3-b900-8e73271f5225.jpg) | ![jonathas-orig-home](https://user-images.githubusercontent.com/36542986/164277484-3ae53deb-c2e0-43e4-b205-0324964deb2e.jpg)